{{cookiecutter.project_slug}}/scripts/verify_pypi_env_variables.py

```diff
@@ -1,36 +1,98 @@
 import os
 import sys
 
-pypi_url = pypi_name = None
+ENV_VAR_PYPI_REPOSITORY_NAME = "PYPI_REPOSITORY_NAME"
+ENV_VAR_PYPI_REPOSITORY_URL_TEMPLATE = "POETRY_REPOSITORIES_{}_URL"
+ENV_VAR_PYPI_REPOSITORY_USERNAME_TEMPLATE = "POETRY_HTTP_BASIC_{}_USERNAME"
+ENV_VAR_PYPI_REPOSITORY_PASSWORD_TEMPLATE = "POETRY_HTTP_BASIC_{}_PASSWORD"
+ENV_VAR_PYPI_REPOSITORY_API_TOKEN_TEMPLATE = "POETRY_PYPI_TOKEN_{}"
 
+pypi_name = "pypi"
+pypi_url = pypi_usename = pypi_password = pypi_api_token = None
+
+exit_code = 0
+
+try:
+    pypi_name = os.environ[ENV_VAR_PYPI_REPOSITORY_NAME]
+except KeyError:
+    pass
+
+print(f"Target pypi repository: '{pypi_name}'")
+
+
+def format_pypi_name(name):
+    return name.replace(".", "_").replace("-", "_").upper()
+
+
+env_var_pypi_repository_url = ENV_VAR_PYPI_REPOSITORY_URL_TEMPLATE.format(
```

Comment:
inverser -> pypi_repository_url_env_var et pareil pour les autres variables

Reply:
Done. Don't know why I did not do it this way in the first place.